packages/one-app-bundler/utils/getExternalFilename.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Returns the external's filename
+ * @param {string} externalName the external name
+ * @returns {string}
+ */
+const getExternalFilename = (externalName) => `${externalName}.js`;
```

Comment:
Can all of these `get*` functions be put into one `utils` file?

Reply:
this function will be deleted